src/csharp/Grpc.Core/NativeDeps.Mac.csproj.include

```diff
@@ -1,13 +1,5 @@
 <Project>
   <ItemGroup>
-    <!-- We are relying on run_tests.py to build grpc_csharp_ext with the right bitness
```

Comment:
I'm rusty this package build... this is not affecting package artifacts though?

Reply:
Nope, this is only for grpc developers to be able to build locally.
The rules for the nuget package that will get applied on net45 are here: https://github.com/grpc/grpc/blob/master/src/csharp/Grpc.Core/Grpc.Core.targets

FTR The packaging of grpc_csharp_ext into the nuget package is here:
https://github.com/grpc/grpc/blob/d100b5cfba3575d466329650137d3c9dd8a75cde/src/csharp/Grpc.Core/Grpc.Core.csproj#L23